docs/development/issue-labeling.md

```diff
@@ -202,16 +186,31 @@ This label is sometimes picked up by tools or websites that try to encourage peo
 
 Add the label `help wanted` to indicate that we need the original poster or someone else to do some work or it is unlikely to get done.
 
-Add a label `logs:problem` to indicate that there's a problem with the logs, and the contributor needs to do one of these things:
+Add a label `auto:bad-vibes` to any discussion containing disrespectful comments such as excessive criticism or ungratefulness.
+
+Add a label `auto:discussion-closed` to close a discussion which had persistent or very bad vibes.
+
+Add a label `auto:discussion-first` to any PR which needs discussing first.
+
+Add a label `auto:format-code` to any Discussion which needs code formatting.
+
+Add a label `auto:logs` to indicate that there's a problem with the logs, and the contributor needs to do one of these things:
 
 1. Provide logs (if there are none yet)
 1. Provide more logs (in case current logs are insufficient)
 1. Format their logs properly
 
-Add a label `needs-code-formatting` to issues with logs/code that needs to be formatted.
+Add a label `auto:needs-code-formatting` to discussions with logs/code that needs to be formatted.
+
+Add a label `auto:needs-details` to discussions which are lacking sufficient details to move forward.
+
+Add a label `auto:no-coverage-ignore` if PR authors have unnecessarily avoided unit tests by istanbul ignoring code.
+
+Add a label `auto:no-done-comments` if PR authors add comments like "Done" to review comments instead of requesting a new review through GitHub's UI.
```

Comment:
The label is a bit confusing, because I also see PR authors "resolve PR comments" by putting done in each comment box, instead of hitting the "resolve" button.

So we have two sets of related behavior:

- pinging reviewers by writing _Done_
- "resolving" PR comments by writing _Done_